lib/zendesk_api/resource.rb

```diff
@@ -146,10 +146,16 @@ def inspect
 
     alias :to_param :attributes
 
+    def attributes_for_save
+      { self.class.singular_resource_name.to_sym => attribute_changes }
+    end
+
```

Comment:
I'm making this method public, mainly because that way I can test it without having to stub the HTTP requests. I reckon it's not a bad idea, it might be useful.